datafusion/common/src/hash_utils.rs

```diff
@@ -392,14 +392,6 @@ pub fn create_hashes<'a>(
                 let array: &FixedSizeBinaryArray = array.as_any().downcast_ref().unwrap();
                 hash_array(array, random_state, hashes_buffer, rehash)
             }
-            DataType::Decimal128(_, _) => {
```

Comment:
Why can this be removed?

Reply:
quoting commit message from the first commit https://github.com/apache/datafusion/pull/13388/commits/1e6994688b8c3eb814336ebe21e43dea79960b30

```
Remove unused code paths from create_hashes

The `downcast_primitive_array!` macro handles all primitive types
and only then delegates to fallbacks. It handles Decimal128 and
Decimal256 internally.
```